docs/research/2026-07-29-shared-workspace-config-injection.md

```
@@ -0,0 +1,162 @@
+## Idea
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)